src/common/function/src/scalars/date/date_add.rs

```diff
@@ -134,7 +135,12 @@ mod tests {
 
         let times = vec![Some(123), None, Some(42), None];
         // Intervals in milliseconds
-        let intervals = vec![1000, 2000, 3000, 1000];
+        let intervals = vec![
+            IntervalDayTime::new(0, 1000),
```

Comment:
Found several such fixes, maybe implement `From<Int32>` for  IntervalDayTime?

Reply:
Maybe not, we don't want to provide this.